content/hi/docs/concepts/architecture/leases.md

```diff
@@ -0,0 +1,102 @@
+---
+title: पट्टों
+content_type: concept
+weight: 30
+---
+
+<!-- overview -->
+
+वितरित प्रणालियों को अक्सर पट्टों की आवश्यकता होती है, जो साझा संसाधनों को लॉक करने और एक सेट के सदस्यों के बीच गतिविधि को समन्वयित करने के लिए एक तंत्र प्रदान करते हैं।
+कुबेरनेट्स में, पट्टे की अवधारणा को [लीज](/docs/reference/kubernetes-api/cluster-resources/lease-v1/) द्वारा दर्शाया गया है।
+`coordination.k8s.io` में ऑब्जेक्ट्स {{< glosary_tooltip text="एपीआई ग्रुप" term_id="api-group" >}},
```

Comment:
Typo in the `glossary_tooltip` shortcode name is resulting [build preview failures](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/657c984d53b6050008327035).

```suggestion
`coordination.k8s.io` में ऑब्जेक्ट्स {{< glossary_tooltip text="API समूह" term_id="api-group" >}},
```